data-raw/jpnprefs.R

```diff
@@ -14,6 +14,10 @@ library(tidyverse)
 # dplyr # 0.7.6
 # tidyr # 0.8.1
 # purrr # 0.2.5
+# (stringr) # 1.3.1
+
+library(polite) # 0.0.0.9004
```

Comment:
Why should we introduce the [polite](https://github.com/dmi3kno/polite) package? 
This package is certainly useful, but has not yet been registered with CRAN.

Reply:
yeah, I suppose it's not entirely necessary to use this package for now. We're only scraping from Wikipedia anyways. I use it as part of my workflow but I understand from a package development/maintenance point of view that it's not necessary. 

We can just replace it with the regular rvest code instead:

```{r}
url <- "https://en.wikipedia.org/wiki/Prefectures_of_Japan"
jpn_pref_raw <- read_html(url) %>% 
    html_nodes("table.wikitable:nth-child(49)") %>% 
    html_table() %>% 
    purrr::flatten_df()

url2 <- "https://en.wikipedia.org/wiki/List_of_Japanese_prefectures_by_population"
jpn_pref2_raw <- read_html(url) %>% 
  html_nodes("table.wikitable:nth-child(7)") %>% 
  html_table() %>% 
  purrr::flatten_df()
```